### ISSUE TITLE
Download broken archive in zip

### ISSUE THREAD

**Issue by ProxyPlay** [Justmilomb/Quixel-Bulk-Downloads#1]
When I use the script to download the assets, it downloads 123 KB archives, although one texture can weigh 50-100 MB, I tried to do it on Google Chrome and Microsoft edge, it doesn't help. It just downloads broken archives with a small size.

**Comment by Justmilomb** [comment]
> When I use the script to download the assets, it downloads 123 KB archives, although one texture can weigh 50-100 MB, I tried to do it on Google Chrome and Microsoft edge, it doesn't help. It just downloads broken archives with a small size.

Thanks for that.
I have tried to troubleshoot today but couldn't figure it out.
I'll give it another crack tomorrow

**Comment by tiandaoyuxi** [comment]
![image](https://github.com/user-attachments/assets/1084c02d-0246-4766-8568-0d574226f942)
The same problem

**Comment by Justmilomb** [comment]
Yeah I can't really fix it but if you look at the forks a guy fixed it.
It takes ages and you are more then likely to get 502 bad gateway since an overload in the network.
